src/Shouldly.Tests/Strings/ShouldBeNullOrEmpty.cs

```diff
@@ -1,24 +1,47 @@
-﻿using Shouldly.Tests.TestHelpers;
+﻿using Xunit;
 
 namespace Shouldly.Tests.Strings
 {
-    public class ShouldBeNullOrEmpty : ShouldlyShouldTestScenario
+    public class ShouldBeNullOrEmpty
     {
-        protected override void ShouldThrowAWobbly()
+        [Fact]
+        public void SingleLetterShouldFail()
         {
-            "a".ShouldBeNullOrEmpty("Some additional context");
+            Verify.ShouldFail(() => 
```

Comment:
@JoeMighty this is what I am thinking for tests going forward.

It tests the error messages with and without source being accessible. Thoughts?